### ISSUE TITLE
minikube cluster not firing up. Please advice.

### ISSUE THREAD

**Issue by mailbagrahul** [kubernetes/minikube#9428]
It was working fine before updating docker desktop & minikube. I tried with new virtual machines but still I'm not able for fire up. Then I tried with hyperv and error log is pasted above. Is this because of version compatibility? I tried all the possible recommendations. I'm waiting for some changes to be moved as soon as possible. Please advice.

λ  minikube start --driver=docker
* minikube v1.11.0 on Microsoft Windows 10 Enterprise 10.0.17763 Build 17763
* Using the docker driver based on user configuration

! 'docker' driver reported an issue: exit status 1
* Suggestion: Restart the Docker service
* Documentation: https://minikube.sigs.k8s.io/docs/drivers/docker/

**Steps to reproduce the issue:** 

1. Docker Desktop up and running along with Kubernetes
2. install new version of minikube
3. minikube start



<!--- TIP: Add the "--alsologtostderr" flag to the command-line for more logs --->
**Full output of failed command:** 
I1008 21:55:16.492096    4544 start.go:98] hostinfo: {"hostname":"VDDP35P-1AUIMAS","uptime":5180,"bootTime":1602214136,"procs":237,"os":"windows","platform":"Microsoft Windows 10 Enterprise"
,"platformFamily":"Standalone Workstation","platformVersion":"10.0.17763 Build 17763","kernelVersion":"","virtualizationSystem":"","virtualizationRole":"","hostid":"fde1d664-1c76-4472-8966-2
d6b83989327"}                                                                                                                                                                                 
W1008 21:55:16.494085    4544 start.go:106] gopshost.Virtualization returned error: not implemented yet                                                                                       
* minikube v1.11.0 on Microsoft Windows 10 Enterprise 10.0.17763 Build 17763                                                                                                                  
I1008 21:55:16.513096    4544 driver.go:253] Setting default libvirt URI to qemu:///system                                                                                                    
W1008 21:55:22.981516    4544 docker.go:99] docker returned error: exit status 1                                                                                                              
* Using the docker driver based on user configuration                                                                                                                                         
I1008 21:55:22.995500    4544 start.go:214] selected driver: docker                                                                                                                           
I1008 21:55:22.995500    4544 start.go:611] validating driver "docker" against <nil>                                                                                                          
I1008 21:55:22.996513    4544 start.go:617] status for docker: {Installed:true Healthy:false Error:exit status 1 Fix:Restart the Docker service Doc:https://minikube.sigs.k8s.io/docs/drivers/
docker/}                                                                                                                                                                                      
                                                                                                                                                                                              
! 'docker' driver reported an issue: exit status 1                                                                                                                                            
* Suggestion: Restart the Docker service                                                                                                                                                      
* Documentation: https://minikube.sigs.k8s.io/docs/drivers/docker/                                                                                                                            
                                                                                                                                                                                              
X Failed to validate 'docker' driver                                                                                                                                                          


**********************************************************************************************************************

The below output is for hyperv driver
**Full output of `minikube start` command used, if not already included:**

λ  minikube start --cpus=2 --memory=3g --driver=hyperv                                                                                                         
* minikube v1.11.0 on Microsoft Windows 10 Enterprise 10.0.17763 Build 17763                                                                                   
* Using the hyperv driver based on user configuration                                                                                                          
* Downloading VM boot image ...                                                                                                                                
    > minikube-v1.11.0.iso.sha256: 65 B / 65 B [-------------] 100.00% ? p/s 0s                                                                                
    > minikube-v1.11.0.iso: 174.99 MiB / 174.99 MiB [] 100.00% 4.71 MiB p/s 37s                                                                                
* Starting control plane node minikube in cluster minikube                                                                                                     
* Downloading Kubernetes v1.18.3 preload ...                                                                                                                   
    > preloaded-images-k8s-v3-v1.18.3-docker-overlay2-amd64.tar.lz4: 526.01 MiB                                                                                
* Creating hyperv VM (CPUs=2, Memory=3072MB, Disk=20000MB) ...                                                                                                 
* Stopping "minikube" in hyperv ...                                                                                                                            
* Powering off "minikube" via SSH ...                                                                                                                          
* Deleting "minikube" in hyperv ...                                                                                                                            
! StartHost failed, but will try again: creating host: create host timed out in 240.000000 seconds                                                             
E1008 21:19:41.888499   15052 main.go:106] libmachine: [stderr =====>] : Hyper-V\Get-VM : Hyper-V was unable to find a virtual machine with name "minikube".   
At line:1 char:3                                                                                                                                               
+ ( Hyper-V\Get-VM minikube ).state                                                                                                                            
+   ~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                    
    + CategoryInfo          : InvalidArgument: (minikube:String) [Get-VM], VirtualizationException                                                             
    + FullyQualifiedErrorId : InvalidParameter,Microsoft.HyperV.PowerShell.Commands.GetVM                                                                      
                                                                                                                                                               
                                                                                                                                                               
* Creating hyperv VM (CPUs=2, Memory=3072MB, Disk=20000MB) ...                                                                                                 
E1008 21:19:44.616573   15052 main.go:106] libmachine: [stderr =====>] : Hyper-V\Get-VM : Hyper-V was unable to find a virtual machine with name "minikube".   
At line:1 char:3                                                                                                                                               
+ ( Hyper-V\Get-VM minikube ).state                                                                                                                            
+   ~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                    
    + CategoryInfo          : InvalidArgument: (minikube:String) [Get-VM], VirtualizationException                                                             
    + FullyQualifiedErrorId : InvalidParameter,Microsoft.HyperV.PowerShell.Commands.GetVM                                                                      
                                                                                                                                                               
                                                                                                                                                               
E1008 21:19:47.589544   15052 main.go:106] libmachine: [stderr =====>] : Hyper-V\Get-VM : Hyper-V was unable to find a virtual machine with name "minikube".   
At line:1 char:3                                                                                                                                               
+ ( Hyper-V\Get-VM minikube ).state                                                                                                                            
+   ~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                    
    + CategoryInfo          : InvalidArgument: (minikube:String) [Get-VM], VirtualizationException                                                             
    + FullyQualifiedErrorId : InvalidParameter,Microsoft.HyperV.PowerShell.Commands.GetVM                                                                      
                                                                                                                                                               
                                                                                                                                                               
E1008 21:19:50.508406   15052 main.go:106] libmachine: [stderr =====>] : Hyper-V\Get-VM : Hyper-V was unable to find a virtual machine with name "minikube".   
At line:1 char:3                                                                                                                                               
+ ( Hyper-V\Get-VM minikube ).state                                                                                                                            
+   ~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                    
    + CategoryInfo          : InvalidArgument: (minikube:String) [Get-VM], VirtualizationException                                                             
    + FullyQualifiedErrorId : InvalidParameter,Microsoft.HyperV.PowerShell.Commands.GetVM                                                                      
                                                                                                                                                               


**Optional: Full output of `minikube logs` command:**
<details>
It was working fine before updating docker desktop & minikube. I tried with new virtual machines but still I'm not able for fire up. Then I tried with hyperv and error log is pasted above. Is this because of version compatibility? I'm waiting for some changes to be moved as soon as possible. Please advice.

</details>

**Comment by medyagh** [comment]
@mailbagrahul does the suggestion in minikube output help ?

``
Suggestion: Restart the Docker service
Documentation: https://minikube.sigs.k8s.io/docs/drivers/docker/
```

can u ensure u have a healthy docker running?

**Comment by mailbagrahul** [comment]
@medyagh - No. I didn't work. Last few days, I have been facing so many issues after upgrading docker-desktop from choco which directly install latest stable version(Docker - 19.03.13) which I'm not able to use docker driver. I tried with hyperV on different VMs but due to firewall/security problem, it is not working. After testing by downgrading to next versions, I found Docker 19.0.8 is the compatible version which uses docker driver by for creating minikube cluster. I don't know why latest version is not compatible to drive with docker driver.

**Comment by mailbagrahul** [comment]
Even docker is running healthy but somehow 19.03.13 with minikube is not working on docker driver. May be it is not able to identify that docker running. no idea what is going on.

**Comment by priyawadhwa** [comment]
Hey @mailbagrahul thanks for your reply. You mentioned:
>I have been facing so many issues after upgrading docker-desktop

are you referring just to minikube, or is docker 19.03.13 giving you issues otherwise?

If you could provide the output of

```
minikube start --driver docker --alsologtostderr
```
that could be helpful as well.

**Comment by medyagh** [comment]
@mailbagrahul  I haven't heard from you, I will close this issue, please feel free to reopen once ready

**Comment by mailbagrahul** [comment]
I have been facing so many issues after upgrading docker-desktop l latest stable version(Docker - 19.03.13) which I'm not able to use docker driver. I tried with hyperV on different VMs but due to firewall/security problem, it is not working.  After testing by downgrading to next versions, I found Docker 19.0.8 is the compatible version which uses docker driverby default  for creating minikube cluster.
